Tests/Unit/MiniAppDownloaderTests.swift

```diff
@@ -247,7 +248,28 @@ class MiniAppDownloaderTests: QuickSpec {
                             break
                         }
                     }
-                    expect(version).toEventually(equal("Mac"), timeout: 20)
+                    expect(version).toEventually(equal(versionId), timeout: 20)
+                }
+            }
+        }
+        describe("mini app downloader") {
```

Comment:
Should we also cover the failure scenario while unzipping ?

Reply:
I totally missed this